Assets/Code/src/Runtime/Match/MatchState.cs

```diff
@@ -17,6 +19,7 @@ public class MatchState : INetworkSerializable {
 
   PlayerState[] playerStates;
   public int PlayerCount { get; private set; }
+  public MatchStateID StateID { get; set; } = MatchStateID.Intro;
```

Comment:
If it doesn't need to be hidden, for these small data objects like this, it's better to just use a public field.

This also should be serialized/deserialized in Serialize/Deserialize below. If you do not know how to do it, leave a TODO for me and I'll get around to doing it.